src/components/Country.js

```diff
@@ -0,0 +1,74 @@
+import styled from 'styled-components';
+
+function Country({ country }) {
+    const { flags, name, population, region, capital } = country || {};
+    const { png } = flags;
+
+    return (
+        <Wrapper>
+            <FlagWrapper>
+                <img src={png} alt={'flag of ' + name} />
+            </FlagWrapper>
+            <TextWrapper>
+                <h2>{name}</h2>
+                <p>
+                    <BoldWrapper>Population:</BoldWrapper>{' '}
+                    {population.toLocaleString()}
+                </p>
+                <p>
+                    <BoldWrapper>Region:</BoldWrapper> {region}
+                </p>
+                <p>
+                    <BoldWrapper>Capital:</BoldWrapper> {capital}
+                </p>
+            </TextWrapper>
+        </Wrapper>
+    );
+}
+
+const Wrapper = styled.div`
+    background-color: var(--elementColor);
+    border-radius: 0.3125em;
+    box-shadow: 0px 0px 7px 2px hsla(0, 0%, 0%, 0.03);
+    justify-self: center;
+    max-width: 20rem;
+    overflow: hidden;
+    transition: background-color 0.3s ease-in;
+`;
+
+const FlagWrapper = styled.div`
+    aspect-ratio: 246/160;
```

Comment:
This aspect ratio seems wrong for the flags.

I think you need something like 0.8.

<img width="749" alt="Screenshot 2022-05-24 at 11 20 12" src="https://user-images.githubusercontent.com/7248028/169997547-4947ed48-c00b-47d1-b28d-d64bff8dfd6b.png">